### ISSUE TITLE
[BUG] Imports of macros should be removed (ESM modules issue)

### ISSUE THREAD

**Issue by vfsfitvnm** [GoogleFeud/ts-macros#48]
**Describe the bug**
Currently, imports of macros are kept, but they should be eliminated.
Other than being a design expectation/requirement (there shouldn't be macros leftovers in the executable code), it also causes a runtime exception.

**Code to reproduce**

The following code

```ts
// macros.ts

// @internal
export function $test() {
    console.log("Test!");
}
```

```ts
// index.ts
import { $test } from "./macros.js";

$test!();
```
gets transpiled to:
```ts
// macros.js

export {};
```

```ts
// index.js
import { $test } from "./macros.js"; // <--- this should have been removed

console.log("Test!");
```
This isn't a problem in CommonJs, but it is in ECMAScript modules (`The requested module './macros.js' does not provide an export named '$test'`).

**Expected behavior**
Imports of macros should be removed!

**Additional context**
NA

**Comment by GoogleFeud** [comment]
Good catch :). All imports of macros (including built-in ones) will be removed.
